### PR TITLE
MDN CSS: Remove external links (for now..)

### PR DESCRIPTION
**Pull request by moollaza**
Unfortunately I misunderstood how External Links work.

Removing this code for now, if Fatheads change to meet my expectations we can bring this back.

Will be added to #311

/cc @hchienjo 

---

https://duck.co/ia/view/mdn_css

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: lib/fathead/mdn_css/parse.pl [hunk 1]
```diff
@@ -139,19 +139,6 @@
     $description = build_abstract( $description, $code, $initial_value );
 
     next unless $title && $link && $description;
-
```
Comment (hchienjo) [inline lib/fathead/mdn_css/parse.pl]: I had began working on this on a new [PR](https://github.com/duckduckgo/zeroclickinfo-fathead/pull/312/commits/aa5371a5015c493bba5ea1cc0b0c112d465ff4ce).

Should I abandon this feature as well over there?

Comment (moollaza) [inline lib/fathead/mdn_css/parse.pl]: @hchienjo yes please, sorry for the extra work

Comment (hchienjo) [inline lib/fathead/mdn_css/parse.pl]: No problem at all.

**Comment by moollaza** [comment]
@hchienjo any opinions on this? 

I tested with DuckPAN  and everything seems to be working as expected 👍

**Comment by hchienjo** [comment]
That's great.

**Comment by hchienjo** [comment]
At least output.txt will not look so clogged as it is now. We can focus on polishing the other features lined up.

**Pull request merged**